### ISSUE TITLE
[Bug Report][3.4.1] Selected option value of v-autocomplete is not announcing on hover by NVDA

### ISSUE THREAD

**Issue by TulsiramJat** [vuetifyjs/vuetify#18814]
### Environment
**Vuetify Version:** 3.4.1
**Last working version:** 3.4.1
**Vue Version:** 3.3.7
**Browsers:** Chrome 119.0.0.0
**OS:** Windows 10

### Steps to reproduce
Select option value then hover over the value

### Expected Behavior
The NVDA screen reader should announce the selected option value when hovered.

### Actual Behavior
The option value does not announce on hover, indicating whether it is functioning, unlike the v-select, which does provide the announcement.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVT8EKgkAQ/ZVlLl5KiYIgLAihPqCgg3rYdJKFdWdZVymif29dE/M0b968mXkvfUNjiuioddi1CDuILdZacouHTDEWd0uutYe+KUhZLhSaHzUoWksFuS20ONKMSX5Huc/gghILm8E02Qn3o3GjNEi4FA8ySvBgwYKEJBleUo9PDorS02ckUw2KKz5504Pbi2qhqiCfDh/iaO5ltO3ouW8v9LHi6C8ufBawDtfhFvq6CVeQfwGVzlhL)

### Other comments
It is working for v-select but not working for v-autocomplete.
